webknossos/webknossos/dataset/layer.py

```diff
@@ -1158,3 +1158,23 @@ def category(self) -> LayerCategoryType:
 
     def _get_largest_segment_id_maybe(self) -> Optional[int]:
         return self.largest_segment_id
+
+    def _get_largest_segment_id(self, view: View) -> int:
+        return np.max(view.read(), initial=0)
+
+    def update_largest_segment_id(
```

Comment:
```suggestion
    def refresh_largest_segment_id(
```

With “update” I would expect a simple setter, maybe refresh is the better name, signaling the semantics and that this is actually an expensive operation?